javascript/exercise1/ex2/aboutme.js

```diff
@@ -0,0 +1,24 @@
+const liText = document.querySelectorAll('li');
+const nickname = document.getElementById('nickname');
+const favorites = document.getElementById('favorites');
+const hometown = document.getElementById('hometown');
+
+// Change the body style so it has a font-family of "Arial, sans-serif".
+for (let i = 0; i < liText.length; i += 1) {
```

Comment:
Should catch liText.length into a variable before running loop to avoid each loop we have to get the length once again